tests/v1/test_api_models.py

```diff
@@ -61,27 +62,74 @@ def test_key_wrapper_with_keyset_json() -> None:
     assert key_wrapper.keyset == PseudoKeyset.model_validate(custom_keyset_dict)
 
 
-def test_pseudo_function() -> None:
-    assert "daead(keyId=ssb-common-key-1)" == str(
+def test_serialize_daead_function() -> None:
+    assert (
+        PseudoFunction(
+            function_type=PseudoFunctionTypes.DAEAD, kwargs=DaeadKeywordArgs()
+        ).__str__()
```

Comment:
Perhaps the string coercion can be called like:
`str(PseudoFunction(<...>))`?
The dunder method `__str__()` is kind of an implementation detail, and one typically uses the user-facing `str()` (which calls `__str__()`)

Reply:
I agree. Fixed!